src/main/java/org/spongepowered/api/entity/living/complex/dragon/phase/EnderDragonPhaseManager.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * This file is part of SpongeAPI, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.api.entity.living.complex.dragon.phase;
+
+import org.spongepowered.api.entity.living.complex.EnderDragon;
+
+/**
+ * An {@link EnderDragon} phase manager.
+ */
+public interface EnderDragonPhaseManager {
+
+    /**
+     * Gets the current dragon phase.
+     *
+     * @return The current dragon phase
+     */
+    EnderDragonPhase getPhase();
+
+    /**
+     * Sets the dragon phase.
+     *
+     * @param phase The new dragon phase
+     */
+    void setPhase(EnderDragonPhaseType phase);
+
```

Comment:
Could you expose a little more to this interface to "pad" it so it's not just looking as though it's a wrapper around what the current phase of the dragon is? Perhaps, adding a getter for the actual `EnderDragon` that is being managed?